### PR TITLE
Added request param to getAll method

### PR DESCRIPTION
**Pull request by JamieBuckell**
Allowed request parameter for use with page and pageSize query options

### CODE REVIEW

File: src/Company.php [hunk 1]
```diff
@@ -262,9 +262,9 @@ protected function init()
      * @return \TeamWorkPm\Response\Model
      * @throws \TeamWorkPm\Exception
      */
-    public function getAll()
+    public function getAll($request = null)
```
Comment (simonschaufi) [inline src/Company.php]: please adjust the docheader as well

Comment (JamieBuckell) [reply src/Company.php]: 👍

Comment (simonschaufi) [reply src/Company.php]: well, null is the default but what is the expected type?

Comment (JamieBuckell) [reply src/Company.php]: updated to match the format in the rest of the project

**Pull request merged**